iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/ExportOptions.java

```diff
@@ -153,6 +152,18 @@ public ExportOptions setVersion(String version) {
         return this;
     }
 
+    public ExportOptions addExtensionVersion(String extensionName, String extensionVersion) {
+        if (extensions != null && !extensions.contains(extensionName)) {
+            throw new PowsyblException(extensionName + " is not an extension you have passed in the extensions list to export.");
+        }
+        extensionsVersions.put(extensionName, extensionVersion);
+        return this;
+    }
+
+    public Optional<String> getExtensionVersion(String extensionName) {
+        return Optional.ofNullable(extensionsVersions.get(extensionName));
+    }
+
```

Comment:
I don't understand how these two methods will be used. Could you explain it to me?

Reply:
`addExtensionVersion ` is to add a particular **extension's version** in which you want the extension to be serialized e.g. if you want to serialize  `CoordinatedReactiveControl` in its 1.0 version, you can use `options.addExtensionVersion("coordinatedReactiveControl", "1.0");`. It is different from the **IIDM-XML version** which is by default 1.1.

`getExtensionVersion` is to get the extension's version in which you want the extension to be serialized if it has been defined (return an empty `Optional` if it has not been defined).

Reply:
Then if for some reason someone calls addExtensionVersion twice with two different versions for the same extension, that extension will be serialized twice, one for each version?

Reply:
Good observation, I didn't think about this case! With the current implementation, the last call to the method is kept... Maybe an exception should be thrown or at least a warning should be written? This case should not happen (the version of an extension is unique for each file/export).

Reply:
I believe an exception would be better than a warning, since this would correspond to an erroneous use of the feature.